pages/common/adb-shell.md

```diff
@@ -0,0 +1,32 @@
+# adb shell
+
+> Android Debug Bridge Shell: Run remote shell command with an Android emulator instance or connected Android devices.
+> More information: <https://developer.android.com/studio/command-line/adb>.
+
+- Start a remote interactive shell in the emulator/device:
+
+`adb shell`
+
+- Reset dangerous permissions to an application:
```

Comment:
I'm not quite sure what you mean here - it's a little unclear. Do you mean `for` instead of `to` here?
```suggestion
- Reset dangerous permissions for an application:
```